requirements.txt

```diff
@@ -0,0 +1,4 @@
+google-api-python-client==1.7.9
```

Comment:
This file looks almost like a copy of antidox/requirements.txt.  Should one of them be removed?

Reply:
Bazel install is via apt-get
https://docs.bazel.build/versions/master/install-ubuntu.html
and cannot be done via pip.

I will update the instructions once the build tree is complete - these steps are necessary so that subsequent changes are all tested via travis.

Reply:
Also, requirements files will be removed once other steps are complete.